pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/SegmentCommitterFactory.java

```diff
@@ -29,18 +29,20 @@
  */
 public class SegmentCommitterFactory {
   private static Logger LOGGER;
-  private final IndexLoadingConfig _indexLoadingConfig;
+  private final TableConfig _tableConfig;
   private final ServerSegmentCompletionProtocolHandler _protocolHandler;
 
-  public SegmentCommitterFactory(Logger segmentLogger, IndexLoadingConfig indexLoadingConfig,
+  public SegmentCommitterFactory(Logger segmentLogger, TableConfig tableConfig,
```

Comment:
My bad, looks like tableConfig is not used, we can remove it.

Reply:
Done.